languages.toml

```
@@ -2421,3 +2421,17 @@ language-server = { command = "nimlangserver" }
 [[grammar]]
 name = "nim"
 source = { git = "https://github.com/aMOPel/tree-sitter-nim", rev = "240239b232550e431d67de250d1b5856209e7f06" }
+
+[[language]]
+name = "just"
+scope = "source.just"
+file-types = ["justfile", "Justfile", ".justfile", ".just"]
```

```suggestion
file-types = ["justfile", "Justfile", "just"]
```

I didn't catch this in the initial PR that added just but the `.`s aren't necessary: the file-types act as either extensions or whole filenames